src/deck/drivers/src/vl53l0x.c

```diff
@@ -1116,7 +1134,3 @@ static const DeckDriver vl53l0x_deck = {
 
 DECK_DRIVER(vl53l0x_deck);
 
-LOG_GROUP_START(range)
-LOG_ADD(LOG_UINT16, range, &range_last)
-LOG_GROUP_STOP(range)
```

Comment:
We should not remove this kind of log variables that could be used by other tools.